doc/developer-notes.md

```diff
@@ -742,14 +743,13 @@ logging messages. They should be used as follows:
   attacker to fill up storage. Note that `LogPrintf(fmt, params...)` is
   a deprecated alias for `LogInfo`.
 
-- `LogError(fmt, params...)` should be used in place of `LogInfo` for
-  severe problems that require the node (or a subsystem) to shut down
-  entirely (e.g., insufficient storage space).
+- `LogError(fmt, params...)` should be used in place of `LogInfo` for severe
+  errors the node admin will need to address (e.g., failure to write data).
```

Comment:
nit: *`instead of`* seems [slightly more common](https://www.italki.com/en/post/question-107941) than *`in place of`* + `that` to streamline readability + `needs to address` to imply urgency:
```suggestion
- `LogError(fmt, params...)` should be used instead of `LogInfo` for severe errors
  that the node admin needs to address (e.g., failure to write data).
```